website/docs/docs/contributing/adapter-development/5-promoting-a-new-adapter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Promoting a new adapter"
+id: "5-promoting-a-new-adapter"
+---
+
+## Model for engagement in the dbt community
+
+The most important thing here is recognizing that people are successful in the community when they join, first and foremost, to engage authentically. 
+
+What does authentic engagement look like? It’s challenging to define explicit rules. One good rule of thumb is to treat people with dignity and respect. 
+
+Contributors to the community should think of contribution *as the end itself,* not a means toward other business KPIs (leads, community members, etc.). Profits are exhausted. Some ways to know if you’re authentically engaging:
+
+- Is an engagement’s *primary* purpose of sharing knowledge and resources or building brand engagement?
+- Imagine you didn’t work at the org you do &mdash; can you imagine yourself still writing this?
+- Is it written in formal / marketing language, or does it sound like you, the human?
+
+## Who should join the dbt community slack
+
+### People who have insight into what it means to do hands-on [analytics engineering](https://www.getdbt.com/analytics-engineering/) work
+
+The dbt Community Slack workspace is fundamentally a place for analytics practitioners to interact with each other &mdash; the closer the users are in the community to actual data/analytics engineering work, the more natural their engagement will be (leading to better outcomes for partners and the community).
+
+### DevRel practitioners with strong focus
+
+DevRel practitioners often have a strong analytics background and a good understanding of the community. It’s essential to be sure they are focused on *contributing,* not on driving community metrics for partner org (such as signing people up for their slack or events). The metrics will rise naturally through authentic engagement.
+
+### Founder and executives who are interested in directly engaging with the community
+
+This is either incredibly successful or not at all depending on the profile of the founder. Typically
+
+### Software Engineers at partner products that are building and supporting integrations with either dbt Core or dbt Cloud
+
+This is successful when the engineers are familiar with dbt as a product or at least have taken our training course
```

Comment:
It seems like these sections were started but never finished.  Is there more here or should we remove them?

Reply:
@jasnonaz did you or I start these thought? I can't remember...

Reply:
```suggestion
### Founder and executives who are interested in directly engaging with the community

This is either incredibly successful or not at all depending on the profile of the founder. Typically, this works best when the founder has a practitioner-level of technical understanding and is interested in joining not to promote, but to learn and hear from users.

### Software Engineers at partner products that are building and supporting integrations with either dbt Core or dbt Cloud

This is successful when the engineers are familiar with dbt as a product or at least have taken our training course. The Slack is often a place where end-user questions and feedback is initially shared, so it is recommended that someone technical from the team be present. There are also a handful of channels aimed at those building integrations, which tend to be a font of knowledge.
```

Reply:
This is great.  Thank you for updating it.